### PR TITLE
sf uploader v2

### PR DESCRIPTION
**Pull request by xj2193**
Clean up the files and update changes to migrate Salesforce to server.

### CODE REVIEW

File: main/daily_update.py [hunk 1]
```diff
@@ -175,9 +178,9 @@ def extract_journey_file(sf_api, sqlfile_name, platform_name):
     :param platform_name: salesforce or pardot
     :return: dataframe
     """
-    fd = open(os.path.abspath(os.path.join(os.pardir, 'sqlserver', sqlfile_name)), 'r')
+    fd = open(os.path.join(settings.path, 'sqlserver/{}'.format(sqlfile_name)), 'r')
     if platform_name == 'salesforce':
-        sqlfile = fd.read().format(settings.OwnerId, settings.PardotUser)
+        sqlfile = fd.read().format(settings.OwnerId, settings.PardotUser, settings.SovereignUser)
```
Comment (djb2188) [inline main/daily_update.py:184]: what is SOVEREIGN user?

File: main/daily_update.py [hunk 2]
```diff
@@ -67,7 +70,7 @@ def extract_contact_file(sf_api, sqlfile_name, platform_name):
     :param platform_name: salesforce or pardot
     :return: dataframe
     """
-    fd = open(os.path.abspath(os.path.join(os.pardir, 'sqlserver', sqlfile_name)), 'r')
+    fd = open(os.path.join(settings.path, 'sqlserver/{}'.format(sqlfile_name)), 'r')
     if platform_name == 'salesforce':
         sqlfile = fd.read().format(settings.OwnerId, settings.PardotUser)
     elif platform_name == 'pardot':
```
Comment (djb2188) [inline main/daily_update.py]: what is happening here. Perhaps comment

File: main/daily_update.py [hunk 3]
```diff
@@ -268,7 +272,15 @@ def main():
...
+# ------------------------- Check if the HealthPro data is well loaded ------------------------------------------------#
 
+    contact_amount = extract_data(engine_sf, 'sf_contact_update.sql').shape[0]
+    while contact_amount > 1000:
+        print('The HealthPro API is still working')
+        time.sleep(600)
+        contact_amount = extract_data(engine_sf, 'sf_contact_update.sql').shape[0]
+    else:
+        print('HealthPro data is well loaded')
 # ------------------------- Extract contacts and journeys from Salesforce and save in local DB ------------------------#
```
Comment (djb2188) [inline main/daily_update.py]: Healthpro is ok if shape / length > 100?

**Comment by djb2188** [comment]
@xj2193 I made a couple of comments / asked questions

File: main/daily_update.py (hunk 1 above)
Comment (xj2193) [reply main/daily_update.py:184]: It's Sovereign CRM. There are some participant journeys whose owner is Sovereign CRM. So the purpose of this sentence is to select all participant journeys with owners (uploader, pardot connector and sovereign)

File: main/daily_update.py (hunk 2 above)
Comment (xj2193) [reply main/daily_update.py]: I added one path parameter in settings.py file which indicate the location where the current parent directory is.

File: main/daily_update.py (hunk 3 above)
Comment (xj2193) [reply main/daily_update.py]: I think 100 and 1000 are both fine.

**Pull request merged**